src/browser_action/vitals.js

```diff
@@ -57,6 +53,16 @@
     }
   });
 
+  function toLocaleFixed({value, unit, precision }) {
+    return value.toLocaleString(undefined, {
+      style: unit && 'unit',
+      unit,
+      unitDisplay: 'short',
+      minimumFractionDigits: precision ?? DEFAULT_UNITS_OF_PRECISION,
+      maximumFractionDigits: precision ?? DEFAULT_UNITS_OF_PRECISION
+    });
```

Comment:
Might want to double check this doesn't suffer from the same issue as #107. String formatters can be annoyingly slow to initialize and this runs in the main thread.

Reply:
Urgh. This is why we can't have nice things!! OK might revert this change (but in Console, HUD and overlay).

Reply:
Other ideas: Log during idle time? Create the formatters once and save them (they may be cached by the browser and this is already functionally happening as in #107).

Reply:
Given only the pop up showed this before and the console log and HUD didn't show it using locale, I'm tempted to just manually format this `X.XXX s`, `XXX ms` or `X.XX`. It's sad that we can't easily support people's preferred locales but I still think consistency and performance is better.

Reply:
https://github.com/GoogleChrome/web-vitals-extension/pull/179